lib/app/models/module/test_case.rb

```diff
@@ -4,15 +4,21 @@ module Inferno
   class Module
     class TestCase
       attr_accessor :id
+      attr_accessor :prefix
       attr_accessor :test_group
       attr_accessor :sequence
       attr_accessor :parameters
 
-      def initialize(id, test_group, sequence, parameters)
+      def initialize(id, test_group, sequence, index, parameters)
         @id = id
         @sequence = sequence
         @test_group = test_group
         @parameters = parameters
+
+        number_to_string = {}
+        ('A'..'ZZZ').each_with_index { |string, string_index| number_to_string[string_index + 1] = string }
+        index_as_string = number_to_string[index]
+        @prefix = "#{test_group.prefix}#{index_as_string}-" unless test_group.prefix.nil?
```

Comment:
How about something like:
```
if test_group.prefix.present?
  index_as_string = 'A'
  index.times { index_as_string.next! }
  @prefix = "#{test_group.prefix}#{index_as_string}-"
end
```

Reply:
This is cleaner, didn't know about `next`.  But I had to add a `.dup` because otherwise I'm mutating a frozen literal, and Rubocop had a recommendation I applied too.